packages/plugin-packages/mashroom-portal-admin-app/src/js/components/PageConfigureDialog.tsx

```diff
@@ -241,7 +241,13 @@ export default class PageConfigureDialog extends PureComponent<Props> {
             const previousTitle = typeof (previousValues.page.title) === 'object' ? previousValues.page.title[this.props.languages.default] : previousValues.page.title;
 
             if (title && title !== previousTitle) {
-                let friendlyUrl = title.replace(/[ -]/g, '_').toLowerCase();
+                let friendlyUrl = title
```

Comment:
I don't like it; it overrides the user input without any warning or hint; also, what about all the special characters of other languages?

Reply:
That's true. URLs with special characters are perfect valid but not very common.
Here is a more generic solution needed for language using non ascii chars to not force users to correct all friendlyUrls derived from the page title.
Rollbacking current change